iceoryx2-ffi/cxx/include/iox2/header_request_response.hpp

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2025 Contributors to the Eclipse Foundation
+//
+// See the NOTICE file(s) distributed with this work for additional
+// information regarding copyright ownership.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Apache Software License 2.0 which is available at
+// https://www.apache.org/licenses/LICENSE-2.0, or the MIT license
+// which is available at https://opensource.org/licenses/MIT.
+//
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+#ifndef HEADER_REQUEST_RESPONSE_HPP
+#define HEADER_REQUEST_RESPONSE_HPP
+
+#include "iox2/unique_port_id.hpp"
+
+namespace iox2 {
+/// Request header used by [`MessagingPattern::RequestResponse`]
+class RequestHeaderRequestResponse {
```

Comment:
I don't like the name, but I wanted to be close to the `HeaderPublishSubscribe`. What do you think about adding a `header` namespace and rename the headers to `PublishSubscribeHeader`, `RequestHeader` and `ResponseHeader`?

Reply:
I really like it!